docs/UserGuide.md

```diff
@@ -11,27 +11,41 @@
 1. **git `2.14`** or later on the command line.
  > Type `git --version` on your OS terminal and ensure that you have the correct version of **git**.
 
-## How to Generate Dashboard
+## How to Generate Report
 1. Download the latest executable Jar on our [release](https://github.com/reposense/RepoSense/releases/latest).
    * Alternatively, you can compile the executable Jar yourself by following our [build from source guide](Build.md).
 1. Execute it on the OS terminal. <br>
 Usage: `java -jar RepoSense.jar -config CSV_CONFIG_FILE_PATH [-output OUTPUT_DIRECTORY] [-since DD/MM/YYYY] [-until DD/MM/YYYY] [-formats FORMAT...]`
 1. The report will be generated in the designated OUTPUT_DIRECTORY, or current working directory otherwise.
-1. To visualize the report, open `index.html`.
-1. If the dashboard was not loaded automatically, upload the `archive.zip` (generated in the OUTPUT_DIRECTORY) manually to load the data.
 
-Sample usage:
+Sample usage to generate the report:
 ```
 $ java -jar RepoSense.jar -config CSV_path.csv -output output_path/ -since 01/10/2017 -until 01/11/2017 -formats java adoc js
 ```
 
+## How to View Dashboard
+# With jar
+1. Ensure that you have generated the report.
+1. Execute it on the OS terminal. <br>
+Usage `java -jar RepoSense.jar -view REPORT_DIRECTORY`
+1. If a `REPORT_DIRECTORY` is entered, a server will be started to display the dashboard, and the report generation will be skipped.
+
+```
+java -jar RepoSense.jar -view report_path/
+```
 Argument List:
 - config : Mandatory. The path to the CSV config file.
 - output : Optional. The path to the dashboard generated. If not provided, it will be generated in the current directory.
 - since : Optional. The start date of analysis. Format: `DD/MM/YYYY`
 - until : Optional. The end date of analysis. Format: `DD/MM/YYYY`
 - formats : Optional. The file formats to analyse. Formats: `alphanumerical file formats`. If not provided, the following file formats will be used. `adoc, cs, css, fxml, gradle, html, java, js, json, jsp, md, py, tag, xml`
+- view : Mandatory. If specified with path to the report directory, all the other arguments will be ignored, and the server will be started to display the dashboard.
```

Comment:
Separate the argument list for generating and viewing of dashboard? In this step, just mention `-view` and transfer the rest of the argument list up with the generating of report section.